lib/monitoring/uptime.ex

```diff
@@ -0,0 +1,73 @@
+defmodule Monitoring.Uptime do
+  require Logger
+
+  @spec monitor_device_uptime(list(map()), integer()) :: :ok
+  def monitor_device_uptime(nodes, timestamp) do
+    {:ok, date_time} = DateTime.from_unix(timestamp, :second)
+
+    {splunk_time, :ok} =
+      :timer.tc(fn ->
+        Enum.each(nodes, fn node ->
+          log_device_status(node, date_time)
+        end)
+      end)
+
+    Logger.info(["arinc_status_to_splunk_ms: ", splunk_time |> div(1000) |> inspect])
```

Comment:
Admittedly, I mainly went with the current approach because other, similar logging in Realtime Signs does the same thing. I agree on the point about readability though, especially for this specific line. For the others, I think it's a bit more subjective on which approach is more readable. I kind of like how the list creates a bit of enumeration for each data whereas if we interpolated all of them, it would look more like just a really long string